features/step_definitions/steps.rb

```diff
@@ -3,6 +3,14 @@
   app.login_as(user: user)
 end
 
+Given("I am logged in as an instance admin") do
+  app.login_as(user: create(:instance_admin))
+end
+
+Given("a project is pending") do
+  app.project_under_test = create(:project, status: :pending)
```

Comment:
@bhaibel - How do you feel about storing the state for the thing under test in the app object?  This way we're not leaking incidental details into Gherkin just to thread the needle?